src/gmock-win32.cpp

```diff
@@ -21,6 +21,40 @@ do {                                    \
 
 namespace {
 
+	typedef DWORD (WINAPI *pfnGetLastError_t)( VOID );
+	pfnGetLastError_t g_pfnGetLastError = nullptr;
+
+	typedef HANDLE (WINAPI *pfnGetCurrentProcess_t)( VOID );
+	pfnGetCurrentProcess_t g_pfnGetCurrentProcess = nullptr;
+
+	typedef BOOL (WINAPI *pfnWriteProcessMemory_t)(HANDLE hProcess, LPVOID lpBaseAddress, LPCVOID lpBuffer, SIZE_T nSize, SIZE_T* lpNumberOfBytesWritten);
+	pfnWriteProcessMemory_t g_pfnWriteProcessMemory = nullptr;
+
+	typedef BOOL (WINAPI *pfnVirtualProtect_t)(LPVOID lpAddress, SIZE_T dwSize, DWORD flNewProtect, PDWORD lpflOldProtect);
+	pfnVirtualProtect_t g_pfnVirtualProtect = nullptr;
+	
+	typedef HMODULE (WINAPI *pfnGetModuleHandleA_t)(LPCSTR lpModuleName);
+	pfnGetModuleHandleA_t g_pfnGetModuleHandleA = nullptr;
+	
+	typedef PVOID(__stdcall *pfnImageDirectoryEntryToDataEx_t)(PVOID Base, BOOLEAN MappedAsImage, USHORT DirectoryEntry, PULONG Size, PIMAGE_SECTION_HEADER *FoundHeader);
+	pfnImageDirectoryEntryToDataEx_t g_pfnImageDirectoryEntryToDataEx = nullptr;
+
+	static bool g_bInit = false;
+	void init() {
+		HMODULE module;
+		module = LoadLibraryA( "kernel32.dll" );
```

Comment:
Could we add error checks for the return values of the `LoadLibraryA` and `GetProcAddress` functions? (to return `false` from `init` in case of fail)